src/components/WalletPopup/WalletTypeSelector.js

```diff
@@ -1,30 +1,55 @@
 import React from 'react';
 import { connect } from 'react-redux';
+import { useTranslation } from 'react-i18next';
 import styled from 'styled-components';
 
 import snxJSConnector, { connectToWallet } from '../../utils/snxJSConnector';
-import { hasWeb3, SUPPORTED_WALLETS, onMetamaskAccountChange } from '../../utils/networkUtils';
+import {
+	hasWeb3,
+	SUPPORTED_WALLETS,
+	SUPPORTED_WALLETS_MAP,
+	onMetamaskAccountChange,
+} from '../../utils/networkUtils';
 import { updateWalletStatus, resetWalletStatus } from '../../ducks/wallet';
 import { toggleWalletPopup } from '../../ducks/ui';
 import { getWalletInfo } from '../../ducks';
 
 import { HeadingMedium } from '../Typography';
 
+import { ReactComponent as CoinbaseWallet } from '../../assets/images/wallets/coinbase.svg';
+import { ReactComponent as LedgerWallet } from '../../assets/images/wallets/ledger.svg';
+import { ReactComponent as MetamaskWallet } from '../../assets/images/wallets/metamask.svg';
+import { ReactComponent as TrezorWallet } from '../../assets/images/wallets/trezor.svg';
+import { ReactComponent as WalletConnect } from '../../assets/images/wallets/walletConnect.svg';
+
+const { METAMASK, LEDGER, TREZOR, COINBASE, WALLET_CONNECT } = SUPPORTED_WALLETS_MAP;
+
+const walletTypeToIconMap = {
```

Comment:
Not a priority but maybe this is something we could move to synthetix-js at some point.